### DISCUSSION TITLE
Auiu (aka RPN) notation ...

### DISCUSSION THREAD

**Discussion by raprism** [uiua-lang/uiua#498] [Language Development]
RPN for calculators is not dead, but a niche. But not only for RPN calculator fans the reversed notation might make more sense to have fun with such a versatile language like Uiua seems to be.

I have to admit that I am quite a novice with Uiua, so my experience is very limited. But I wonder if there are serious limitations, if one would introduce an _Auiu_ mode `;-)`

**Comment by Omnikar** [comment]
If you write functions separated by `;`, they will be flipped when formatted. This allows you to write things from left to right, RPN-style. For instance, if I want to push `4` and `9`, then multiply them, then square root the result, I can write these instructions left to right like `4 9;mul;sqrt` and it will format to `√×4 9`, which does the operations in the order I wanted.

**Reply by prismv** [reply]
Good to know. Thank you.
